shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -42,7 +240,19 @@ impl CassandraConnection {
             context.build()
         });
         match self {
-            CassandraConnection::Datastax { schema_awaiter, .. } => {
+            #[cfg(feature = "cassandra-cpp-driver-tests")]
+            Self::Datastax { schema_awaiter, .. } => {
+                *schema_awaiter = Some(
+                    SessionBuilderScylla::new()
```

Comment:
why dont we first get a mutable reference to schema_awaiter so that we dont have to duplicate this logic?